### DISCUSSION TITLE
Why the homepage navigation button not working?

### DISCUSSION THREAD

**Discussion by luofeicq** [adityatelange/hugo-PaperMod#726] [Q&A]
When I set the baseurl to empty  as “” , everything worked fine on local services. However, after  I deployed my site on netlify , once I  clicked any links to view the subpages,  I can not get back to the homepage by click the “home” or logo which was shown on the left top of the pages as the navigation button.

I was using the “page info ”style.

update: I found it worked fine on "git pages". I found that when I deployed the website to gitpage, the address displayed in the browser address bar had an additional secondary address:

[https://luofeiblog.netlify.app/](https://luofeiblog.netlify.app/)
[https://luofeicq.github.io/luofeiblog/](https://luofeicq.github.io/luofeiblog/)

**Comment by adityatelange** [comment]
Hugo recommends setting a `baseURL: https://luofeiblog.netlify.app/`

**Reply by luofeicq** [reply]
> Hugo recommends setting a `baseURL: https://luofeiblog.netlify.app/`

Thank you for your reply. The issue has been solved by rebuild the site. I set the baseurl of my new site to “/“ and everything work fine now. However, I still think that there might be some conflict in the config yaml which caused the issue of the old site.

**Comment by jermanuts** [comment]
@adityatelange Hello, I'm facing the same issue and tried changing my baseURL multiple times and still doesn't work

https://github.com/jermanuts/blog/blob/953d38a9e7483b55ac5955b8f084cb39371fe120/config.yml#L1C41-L1C41

If I visit any page, I can't return to my homepage.

EDIT: I fixed it by running hugo after updating my baseURL. 
If you ran hugo before adding your site to baseURL, for example at the first time setting it up, you probably had baseURL set to blank "". So after getting everything set up and add your site to baseURL, run hugo command.
